minio/minio
# Capture percentage of cpu load and memory used

## Community Contribution License

All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description

By default the cpu load is the cumulative of all cores. Capture the percentage load (load * 100 / cpu-count)

Also capture the percentage memory used (used * 100 / total)

## Motivation and Context

Will be useful for tooling

## How to test this PR?

- Run `mc admin prometheus metrics <alias> resource`
- Verify that the output contains following new metrics
  - minio_node_mem_used_perc
  - minio_node_mem_used_perc_avg
  - minio_node_mem_used_perc_max
  - minio_node_cpu_avg_load1_perc
  - minio_node_cpu_avg_load1_perc_avg
  - minio_node_cpu_avg_load1_perc_max
  - minio_node_cpu_avg_load5_perc
  - minio_node_cpu_avg_load5_perc_avg
  - minio_node_cpu_avg_load5_perc_max
  - minio_node_cpu_avg_load15_perc
  - minio_node_cpu_avg_load15_perc_avg
  - minio_node_cpu_avg_load15_perc_max
- Verify that the values of these fields are calculated correctly

## Types of changes

## Checklist:
